manifests/init.pp

```diff
@@ -21,5 +21,36 @@
     package_ensure => $package_ensure,
   }
 
+  # Create jobs from hiera
+  $cron_job=hiera_hash('cron::job','')
+  if $cron_job != '' {
+    create_resources('cron::job',$cron_job)
+  }
+  
+  $cron_job_multiple=hiera_hash('cron::job::multiple','')
+  if $cron_job_multiple != '' {
```

Comment:
Can you change these checks so that hiera_hash defaults to undef and adjust the `if`? E.g.:
```
  $cron_job=hiera_hash('cron::job', undef)
  if $cron_job {
```